ocs_ci/ocs/machine.py

```diff
@@ -263,6 +266,26 @@ def create_custom_machineset(
                     'tags'
                 ][0]['name'] = f"kubernetes.io/cluster/{cls_id}"
 
+                # if we have infra nodes to be added then we need
+                # taint and extra label
+                if role == 'infra':
+                    taint_dict = {
```

Comment:
once more thing - this is quite similar to https://github.com/red-hat-storage/ocs-ci/blob/5dd1743cff414d2de5555b2d20f30cfcb64511c9/ocs_ci/deployment/deployment.py#L234

Should we maybe create a common function and call it from both places?

Reply:
The difference I see between those usages is 
1. In one case , we already have nodes and we are just applying lables 
2. Another case which this PR has the code, we create the instances using machinesets whose resource yaml already has the labels.

I am not sure if creating the machineset first and then applying the label and taints has same effect as creating machineset along with labels and taints ? Please clarify.

Reply:
oh I just meant that maybe we can move the code which is common to both places to be in a separated function

Reply:
not a big deal, feel free to ignore